src/mol-model/structure/structure/element/loci.ts

```diff
@@ -148,7 +148,7 @@ export namespace Loci {
      * The loc argument of the callback is mutable, use Location.clone() if you intend to keep
      * the value around.
      */
-    export function forEachLocation(loci: Loci, f: (loc: Location) => any) {
+    export function forEachLocation(loci: Loci, f: (loc: Location) => void) {
```

Comment:
Why the `void` here? I think it's ok if the lambda returns a value if we just discard it.